app/controllers/legacy/sessions_controller.rb

```diff
@@ -285,6 +288,22 @@ def email_usernames
 
     private #################
 
+    def save_new_params_in_session
+      # Store these params in the session so they are available if the lookup_login
+      # fails.  Also these methods perform checks on the alternate signup URL.
+      set_client_app(params[:client_id])
+
```

Comment:
it looks weird to have lines between these related statements.  no big deal of course.

Reply:
You're right! Fixed.